content/docs/reference/reference.json

```diff
@@ -129,14 +129,6 @@
     "short_description": "Certificate relative file location string",
     "services": []
   },
-  "client-certificate-authority": {
```

Comment:
I think it would be better to remove this entry in a separate PR, if this PR will be backported to the v0.25 docs site. The corresponding change in Core (https://github.com/pomerium/pomerium/pull/4918) won't be released until v0.26 at the earliest.

Reply:
@kenjenkins thanks for letting me know. I don't see a purpose in backporting these changes unless we want the idp fragments to be available in v25 for some reason?

Reply:
It's nice for the reference table entries to link directly to the corresponding setting documentation, but I agree it's not a major issue, so if you want to skip backporting this PR I think that would be fine as well.